posthog/storage/object_storage.py

```diff
@@ -270,6 +270,7 @@ def object_storage_client() -> ObjectStorageClient:
                     signature_version="s3v4",
                     connect_timeout=1,
                     retries={"max_attempts": 1},
+                    s3={"addressing_style": "path"},
```

Comment:
is it this change that has switched us from host.docker.internal to seaweedfs in the snapshots

why did we need to change it?